tests/nested/manual/split-refresh/task.yaml

```diff
@@ -0,0 +1,224 @@
+summary: Check app refreshes don't wait for a reboot in hybrid systems
+
+details: |
+  Check that (on hybrid systems) refreshes of both essential (e.g., kernel,
+  gadget) and non-essential snaps don't make apps and non-model bases wait for
+  a reboot like the essential snaps. Check that if an app depends on the model
+  base, it does wait for its refresh, and consequently, a reboot.
+
+systems: [ubuntu-22.04-64, ubuntu-24.04-64]
+
+environment:
+    NESTED_ENABLE_TPM: false
+    NESTED_ENABLE_SECURE_BOOT: false
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+    NESTED_ENABLE_OVMF: true
+    NESTED_REPACK_KERNEL_SNAP: true
+    # store related setup
+    STORE_ADDR: localhost:11028
+    STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+  if [ "$TRUST_TEST_KEYS" = "false" ]; then
+      echo "This test needs test keys to be trusted"
+      exit
+  fi
+  echo "Install used snaps"
+  snap install test-snapd-curl --devmode --edge
+  snap install jq --devmode --edge
+  if [ -d /var/lib/snapd/seed ]; then
+      mv /var/lib/snapd/seed /var/lib/snapd/seed.orig
+  fi
+  "$TESTSTOOLS"/store-state setup-fake-store "$STORE_DIR"
+
+
+restore: |
+  if [ "$TRUST_TEST_KEYS" = "false" ]; then
+      echo "This test needs test keys to be trusted"
+      exit
+  fi
+  rm -rf /var/lib/snapd/seed
+  if [ -d /var/lib/snapd/seed.orig ]; then
+      mv /var/lib/snapd/seed.orig /var/lib/snapd/seed
+  fi
+  "$TESTSTOOLS"/store-state teardown-fake-store "$STORE_DIR"
+  rm -rf ./classic-root
+
+debug: |
+  if remote.exec true; then
```

Comment:
Can you disable `set -e` around the debug section so that debugging "failures" do not break the testbed?

Reply:
Is that the case? My understanding is that errors during debugging only stop the debugging script